vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -4,182 +4,198 @@ sidebarDepth: 2
+      ```
+      e.g. `aws eks --region us-east-2 update-kubeconfig --name cluster-1`
+3. Your current context should now be configured for your AWS cluster. Run the command below to check:
     ```sh
     kubectl config current-context
     ```
    Your output should look something like this:
     ```
     arn:aws:eks:us-east-2:483173223614:cluster/cluster-1
     ```
+ For more details and troubleshooting, see [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
 
 ### Install the NGINX Ingress Controller
-1. Add the NGINX controller for ingress. This depends on your role having permissions for ELB.
-   - For basic NGINX ingress install run this command
-    ```sh
-    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.2/deploy/static/provider/cloud/deploy.yaml
-    ```
-   - See <https://kubernetes.github.io/ingress-nginx/deploy/#aws> as well as [this](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more detailed install steps.
-2. Get the ELB external URL for your NGINX install
-   - Run: ```kubectl get services -n ingress-nginx```
-   - Get the value of the external address (EXTERNAL-IP) for the ingress-nginx-controller:
-```
-NAME                                 TYPE           CLUSTER-IP       EXTERNAL-IP                        
-ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1ff4dadb44639a6bbf707e-0e0a483d966405ee.elb.us-east-2.amazonaws.com
+Add the NGINX controller for the ingress. This depends on your role having permissions for ELB.
+
+   - From the EKS cloud shell (or AWS client), use the login:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.5/deploy/static/provider/cloud/deploy.yaml 
 ```
+See the K8s [Load Balancer Install Guide for AWS](https://kubernetes.github.io/ingress-nginx/deploy/#aws) and the [NGINX as Ingress Controller on EKS](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more details.
 
 ::: tip
-NGINX is working correctly if a `404 Not Found` error page is generated when accessing the EXTERNAL-IP from your browser. Alternatively, you can [set up a simple test application](../devops/manage-nginx.md#verify-the-nginx-ingress-install) using your local `kubectl`. You can also [customize the NGINX ingress](../devops/manage-nginx.md#customize-the-nginx-configuration) to optimize the configuration for Entando.
+NGINX is working correctly if a `404 Not Found` error page is generated when accessing the EXTERNAL-IP from your browser. You can [set up a simple test application](../devops/manage-nginx.md#verify-the-nginx-ingress-install) using your local `kubectl`. You can also [customize the NGINX ingress](../devops/manage-nginx.md#customize-the-nginx-configuration) to optimize the configuration for Entando.
```

Comment:
"alternatively" means something here. either you can check the URL or setup a test application. you don't need to do both.